src/robusta/core/reporting/blocks.py

```diff
@@ -76,6 +76,31 @@ def __init__(self, filename: str, contents: bytes):
         """
         super().__init__(filename=filename, contents=contents)
 
+    def truncate_content(self,  max_file_size_bytes: int) -> bytes:
+        """
+        Truncates the log file by removing lines from the beginning until its size is within the given limit.
+        """
+        content_length = len(self.contents)
+
+        if content_length <= max_file_size_bytes:
+            return self.contents
+
+        lines = self.contents.decode('utf-8')
+        lines = lines.splitlines()
+        byte_length_newline = len('\n'.encode('utf-8'))
+
+        truncated_lines: List[str] = []
+        for idx, line in enumerate(lines):
+            line_content_length = len(line.encode('utf-8'))
```

Comment:
you could replace this with one line 
`line_content_length = len(line.encode('utf-8')) + byte_length_newline`